app/src/main/res/layout/lessons_not_started_chapter_view.xml

```diff
@@ -13,26 +13,25 @@
   </data>
 
   <LinearLayout
+    android:id="@+id/not_started_chapter_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:onClick="@{() -> viewModel.onClick(viewModel.explorationId)}"
     android:orientation="vertical">
 
     <LinearLayout
-      android:id="@+id/not_started_chapter_container"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:background="@drawable/border_white"
-      android:clickable="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? true : false}"
+      android:background="@drawable/chapter_white_bright_green_border_background"
       android:minHeight="48dp"
-      android:onClick="@{() -> viewModel.onClick(viewModel.explorationId)}"
       android:orientation="horizontal">
 
       <TextView
         android:id="@+id/chapter_index"
         android:layout_width="60dp"
         android:layout_height="match_parent"
         android:layout_marginStart="0dp"
-        android:background="@drawable/border_bright_green"
+        android:background="@drawable/chapter_dark_green_bright_green_border_background"
```

Comment:
```suggestion
        android:background="@drawable/chapter_dark_green_fill_bright_green_border_background"
```

WDYT about this name, instead, or something similar
? The double color in a row part of the name is slightly confusing at the moment. Ditto for the others.

Reply:
done